src/components/Thanks/ShareStepper.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+	<div class="tw-my-2 tw-px-2">
+		<div style="width: 136px;" class="tw-mx-auto tw-px-1">
+			<div class="tw-flex tw-items-center">
+				<template v-for="step in steps">
+					<ShareStepperStep
+						:key="step.key"
+						:step="step.key"
+						:is-last-step="step.key === steps.length - 1"
+						:text="step.text"
+					/>
+				</template>
+			</div>
+		</div>
+		<p class="tw-text-center tw-mt-4 tw-text-h3">
+			{{ lenderName }}, complete your support by sharing this loan.
```

Comment:
The name should be masked from fullstory with `<span class="fs-mask">{{ lenderName }}</span>`